src/utils/jobData.ts

```diff
@@ -0,0 +1,15 @@
+// TODO: Add real job data
```

Comment:
Instead of creating a utils folder, let's either:

1. Move this to the `routes/JobBoard`
2. We could consider creating a `data` folder.

It might also make sense to make this file pure JSON rather than TS